json-rpc/README.md

```diff
@@ -322,13 +396,152 @@ execution that violates the invariant. This field encodes a trace in the [ITF Fo
 
 **Example**:
 
+Execute the following command:
+
 ```sh
-curl -X POST http://localhost:8822/rpc \
+$ curl -X POST http://localhost:8822/rpc \
   -H "Content-Type: application/json" \
   -d '{"jsonrpc":"2.0","method":"checkInvariant","params":{"sessionId":"1","invariantId":0},"id":3}'
+```
+
+The output is as follows:
+
+```json
 {"jsonrpc":"2.0","id":5,"result":{"sessionId":"1","invariantStatus":"SATISFIED"}}
 ```
 
+### 2.6. Method query
+
+Given a session identifier, query the current context for values of several kinds:
+
+ - `"TRACE"`: A concrete trace that follows the symbolic path constructed so far.
+   There may be multiple such traces. This call returns the trace that
+   was found by the SMT solver.
+ - `"VIEW"`: The value of the view operator. The name of this operator must be
+   specified in the `view` field of the `loadSpec` method. There may be multiple
+   such values. This call returns the view of the model that was found by the
+   SMT solver.
+ - TBD: More kinds to be added in the future.
+
+**Precondition.** This method is successful only if the current context has a
+model. Otherwise, the result is undefined. Below are the conditions under which the
+model is guaranteed to exist:
```

Comment:
Should we check for these and return an error?

Seems like something that's easy to mess up and result in unexpected behavior.